mlflow/server/js/src/experiment-tracking/components/MetricsPlotPanel.js

```diff
@@ -63,6 +65,8 @@ export class MetricsPlotPanel extends React.Component {
   // a single-click event.
   SINGLE_CLICK_EVENT_DELAY_MS = this.MAX_DOUBLE_CLICK_INTERVAL_MS + 10;
 
+  DURATION_BETWEEN_HISTORY_UPDATES_MS = 8000;
```

Comment:
@cedkoffeto Is there any specific reason that you chose 8000?

Reply:
Hi @harupy  
Since the refresh of the plot was not as interesting to see, 5 or 3 seconds was a little too fast so I increased to 8 because 10 was also a little too long as the waiting time so I was trying to find a trade-off

but now with your version, the refresh seems to be way better so there is no need to have a high delay.

Hope it makes sense

Reply:
I think this does depend on how often you log metrics. For example, when you train a large deep learning model and logs metrics by epoch, 3 seconds might be too short.

Reply:
@cedkoffeto  I do agree with you 
So we can keep 3 or increase to 5 to handle quick updates cases (3s is ok for me btw).